lib/nerves/artifact/build_runners/docker.ex

```diff
@@ -64,7 +64,20 @@ defmodule Nerves.Artifact.BuildRunners.Docker do
 
   @version "~> 1.12 or ~> 1.12.0-rc2 or >= 17.0.0"
 
-  @working_dir "/nerves/build"
+  # nerves_system_br v1.8.0 changed to a Docker image that moves the
+  # project build directory and runs everything as a non-root user.
+  defp new_docker_image?() do
+    version_c = Application.spec(:nerves_system_br)[:vsn]
+    version_c != nil and Version.match?(to_string(version_c), "~> 1.28")
```

Comment:
```suggestion
    version_c != nil and Version.match?(to_string(version_c), ">= 1.28")
```
I think this should cover all 1.x versions but do we can to make it any version higher than 1.28?

Reply:
You mean cover 2.0 and higher? I'm totally for your change, but I think I'm misunderstanding the reason.

Reply:
Ya, that's what I mean. All versions 1.28 and higher including 2.0